workflow/metrics/test_exporter.go

```diff
@@ -139,6 +139,23 @@ func (t *TestExporter) getNamedFloat64GaugeData(name string, attribs *attribute.
 	return nil, fmt.Errorf("%s type gauge[float64] not found in %v", name, mtc)
 }
 
+func (t *TestExporter) getNamedInt64GaugeData(name string, attribs *attribute.Set) (*metricdata.DataPoint[int64], error) {
+	mtc, err := t.getNamedMetric(name)
```

Comment:
isn't this usually `m`? vs. `mtc` which feels like a new abbreviation

Reply:
No, this isn't the same thing as the usual `m`, which is usually the "global metric object". This is a specific, opentelemetry internal, obejct type.

The difference in naming appears to have worked successfully here then.

Reply:
This kind of thing is better reviewed in #13265 really

Reply:
Yea I noticed that some style originates from the first PR, but it is easier to understand as smaller bits 😅 
I'm getting to that one, I just need a lot larger chunk of time for it given how large it is (it also still doesn't adhere to the docs style guide per https://github.com/argoproj/argo-workflows/pull/13232#pullrequestreview-2133572454).